tools/testbench/file.c

```diff
@@ -710,9 +766,18 @@ static int file_process(struct processing_module *mod,
 
```

Comment:
side note - the "TODO: duplex mode" above - shouldn't it terminate the function as long as duplex isn't supported?